lib/src/widget/button.dart

```diff
@@ -181,6 +183,8 @@ class _NeumorphicButtonState extends State<NeumorphicButton> {
   }
 
   Widget _build(BuildContext context) {
+    final appBarPresent = NeumorphicAppBarTheme.of(context) != null;
```

Comment:
@florent37 I have a problem here :/ my idea was that if the button is under an appBar, we can use the default style of the app bar. So far so good, but the problem is that the code here is a bit complex around the initialStyle. Any direction you can give me ? I have hack quickly to see if my solution was working and it works great ! But now I need to integrate with your initialStyle code. Any help will be appreciated ^^

Reply:
So I have something working here now, but if you check the example I added, not all buttons are getting the right app bar theme ! Any help will be appreciated ^^ except this my PR is now go to review.

Reply:
ok found the problem ! All good now ! And with your other PR #134 it will also give the ability to configure the shape :) nice ! 

Ready to be reviewed